lib/submission.rb

```diff
@@ -0,0 +1,32 @@
+require 'dotenv/load'
+require 'twilio-ruby'
+
+class Submission
+  def initialize(order = Order.new)
```

Comment:
What's the benefit of having a default to start a new order when you submit the previous order? Sorry if I'm missing something.

Reply:
dependency injection, or I could be doing it wrong...